### DISCUSSION TITLE
Socket IO Redis Adapter for python socket-io

### DISCUSSION THREAD

**Discussion by ch-dev401** [miguelgrinberg/python-socketio#1397] [Q&A]
I just want to ask on how I could connect the socket connections via rooms, both the socket-io javascript and python-socket-io. Having a hard time connecting them, I could still not be able to listen in any messages from javascript app, also could not send message from fastapi app to the javascript app

**Comment by miguelgrinberg** [comment]
I suggest you look at the examples in this repository for guidance. For example, https://github.com/miguelgrinberg/python-socketio/blob/main/examples/server/asgi/app.py uses rooms.
